src/proof/proof_node_updater.h

```diff
@@ -67,6 +67,17 @@ class ProofNodeUpdaterCallback
                       const std::vector<Node>& args,
                       CDProof* cdp,
                       bool& continueUpdate);
+
+  /** As above, but at post-visit. */
```

Comment:
Maybe comment on why there is no `continueUpdate`: "We do not pass a `continueUpdate` flag to this method or the one below, since the children of this proof node have already been updated."